src/controllers/events.cr

```diff
@@ -94,6 +94,7 @@ class Events < Application
     placeos_client = get_placeos_client
 
     host = input_event.host || user.email
+    head :forbidden unless host == user.email || get_user_calendars.find { |cal| cal.id == host }
```

Comment:
Case sensitivity of this comparison is likely to cause issues. From a quick look, it's an issue that is more widespread through this controller. If not addressed here, definitely worth addressing separately.

Reply:
ahh no I made this check just then, all the rest go through a shared check that does downcase. I'll update this

Reply:
Ah yep, lots of other state I missed. May be a good example to strengthen the argument for lifting emails to a type of their own (as suggested by @caspiano in https://github.com/PlaceOS/models/issues/108#issuecomment-897277771). This would then allows comparison to defined once and used everywhere.